app/services/profiles/extract_data.rb

```diff
@@ -0,0 +1,47 @@
+module Profiles
+  module ExtractData
+    DIRECT_ATTRIBUTES = %i[
+      available_for
+      behance_urL
```

Comment:
is this case sensitive? because the `L` is here capitalized

Reply:
Didnt want you to mistake it for an "i" 😂

Reply:
Thanks @rhymes, fixed now.